### PR TITLE
Key rev value & make (remaining) tests pass

### PR DESCRIPTION
**Pull request by gwicke**

### CODE REVIEW

**Comment by earldouglas** [comment]
Yay!

File: config.example.yaml [hunk 1]
```diff
@@ -60,6 +60,11 @@ templates:
             version: 1.0.0
             type: file
 
+      /{api:sys}/{module:key_rev_value}: &wp/sys/key_value
```
Comment (d00rman) [inline config.example.yaml]: The `wp/sys/key_value` achor name is already in use. We should probably rename that to `wp/sys/key_rev_value` for consistency's sake, anyway.

File: mods/key_rev_value.js [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+        return { status: 400 };
+    });
+};
+
+module.exports = function(options) {
+    var krvBucket = new KRVBucket(options);
+
+    return {
+        spec: spec, // Re-export from spec module
+        test: null, // Spec test function from spec module
```
Comment (earldouglas) [inline mods/key_rev_value.js]: Is the optional `test` function meant for tests that are hard to write in terms of specs and `x-ample`s?

**Comment by earldouglas** [comment]
Looks good to me.

File: mods/key_rev_value.js (hunk 2 above)
Comment (earldouglas) [inline mods/key_rev_value.js]: Chatted with @d00rman ; sounds like this is a way to add implementation-specific tests that augment any spec tests that might be in place.  :+1:

Comment (gwicke) [inline mods/key_rev_value.js]: I think we'll actually not need to re-export any tests, as

- spec tests are exported by the spec module, and
- implementation-specific tests can simply be run by the module tests.

**Pull request merged**